src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE configuration>
+
+<configuration scan="true">
+
+    <conversionRule conversionWord="clr" converterClass="org.springframework.boot.logging.logback.ColorConverter"/>
+    <conversionRule conversionWord="wex" converterClass="org.springframework.boot.logging.logback.WhitespaceThrowableProxyConverter"/>
+    <conversionRule conversionWord="wEx" converterClass="org.springframework.boot.logging.logback.ExtendedWhitespaceThrowableProxyConverter"/>
+
+    <property resource="META-INF/build-info.properties"/>
+    <property name="LOG_LOCATION" value="./logs"/>
+    <property name="C_LOG_PATTERN" value="${C_LOG_PATTERN:-%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd HH:mm:ss.SSSXXX}} %clr(%5p) %clr([%X{traceId:-},%X{spanId:-}]){magenta} %clr(-&#45;&#45;){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %clr([%17X{stepType}]){blue} %X{requestTo:-} %X{requestContent:-} %X{responseContent:-} %X{responseCode:-} %X{responseInMs:-} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"/>
```

Comment:
Please use `tab` instead of just a single space (` `) to delimit values. Need it when grokking, grepping etc these files.

Reply:
replaced spaces with tabs and removed fixed width of the step type. New logs:
```
1655893460	0.0.1-SNAPSHOT	 INFO	0:0:0:0:0:0:0:1	incoming.request	[60e6881badb43cd8,60e6881badb43cd8]	-	-	-	-	-	:	Request received for configuration: get-message
1655893460	0.0.1-SNAPSHOT	 INFO	0:0:0:0:0:0:0:1	assign	[60e6881badb43cd8,f45dfea60ab8b1a2]	-	-	-	-	1	:	Executed: step_1
1655893460	0.0.1-SNAPSHOT	 INFO	0:0:0:0:0:0:0:1	assign	[60e6881badb43cd8,6221d639ccf259a6]	-	-	-	-	0	:	Executed: step_2
1655893460	0.0.1-SNAPSHOT	 INFO	0:0:0:0:0:0:0:1	assign	[60e6881badb43cd8,d59d530829fd81c1]	-	-	-	-	0	:	Executed: step_3
1655893461	0.0.1-SNAPSHOT	ERROR	0:0:0:0:0:0:0:1	return	[60e6881badb43cd8,60e6881badb43cd8]	-	-	-	-	-	:	Error: step_4
1655893461	0.0.1-SNAPSHOT	ERROR	0:0:0:0:0:0:0:1	incoming.response	[60e6881badb43cd8,60e6881badb43cd8]	-	-	-	-	-	:	Error executing configuration: get-message

```